mongodbatlas/resource_mongodbatlas_privatelink_endpoint_service.go

```diff
@@ -104,8 +108,8 @@ func resourceMongoDBAtlasPrivateEndpointServiceLinkCreate(d *schema.ResourceData
 	}
 
 	stateConf := &resource.StateChangeConf{
-		Pending:    []string{"NONE", "PENDING_ACCEPTANCE", "PENDING", "DELETING", "WAITING_FOR_USER"},
-		Target:     []string{"AVAILABLE", "REJECTED", "DELETED"},
+		Pending:    []string{"NONE", "INITIATING", "PENDING_ACCEPTANCE", "PENDING", "DELETING", "WAITING_FOR_USER"},
+		Target:     []string{"AVAILABLE", "REJECTED", "DELETED", "FAILED"},
```

Comment:
Related to the docs comment I made and fix you did in the last commit, do we have states here that are no longer applicable?

Reply:
Good eye, I need to delete the `WAITING_FOR_USER` pending string since it doesn't apply this situation, for `DELETED` it's harcoded for refresh function, will make changes, thank you melissa